### PR TITLE
Feature/cmp migration

### PR DESCRIPTION
**Pull request by berkayinceisci**
share UI across multi-platform

### CODE REVIEW

**Review by baseru — COMMENTED** [review]
We are aware that using Jetpack Compose on the iOS side will result in a huge bundle size. However, we are building these apps for demo purposes, so the ease of development is more important for us.

**Comment by baseru** [comment]
@inceisciberkay Did you try initializing the Lokate SDK on the demo app? I am looking through my phone and cannot sure if it is added anywhere else but it looks like you removed the startup initializer from the application manifest, without the startup initializer LokateSDK will probably crash due to not handling SDK initalization manually.

**Comment by berkayinceisci** [comment]
> @inceisciberkay Did you try initializing the Lokate SDK on the demo app? I am looking through my phone and cannot sure if it is added anywhere else but it looks like you removed the startup initializer from the application manifest, without the startup initializer LokateSDK will probably crash due to not handling SDK initalization manually.

Yes, it is moved from kmmsdk/src/androidMain/AndroidManifest.xml to composeApp/src/androidMain/AndroidManifest.xml. Actually, even though SDK is initialized without any problem, I cannot handle its output as expected for now, which means the branch is not ready to merge (hopefully tomorrow will be). I just opened the merge request to see diff section.

File: composeApp/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            kotlinOptions {
+                jvmTarget = "11"
+            }
+        }
+    }
+    
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
```
Comment (baseru) [inline composeApp/build.gradle.kts:21]: We can remove iosX64 and iosSimulatorArm64 targets from both the library and the demo apps. iOS Simulator is not capable of accessing the Mac's Bluetooth, and I believe iosx64 is a target for intel based Mac's ios simulation.

Comment (berkayinceisci) [reply composeApp/build.gradle.kts:21]: I need iOS simulator to see if the components are rendered correctly even though its bluetooth functionality does not work (I don't have an access to an iphone). But we can remove iosx64

**Pull request closed without merging**